common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1106,8 +1115,9 @@ def test_rescore_problem_additional_correct(self):
         self.assertEqual(result['success'], 'incorrect')
         self.assertEqual(module.get_score(), (0, 1))
         self.assertEqual(module.correct_map[answer_id]['correctness'], 'incorrect')
-        # Expect that the number of attempts is incremented
+        # Expect that the number of attempts is incremented, still same attempt
```

Comment:
This one is just first attempt (not still same)

Reply:
@jolyonb Does your comment mean the code comment is confusing? 
It's really confusing to me when your PR comment contradicts the code comment you made.

Reply:
The comment here is confusing; a complete comment would be "Expect that the number of attempts has incremented from 0 to 1, and that the context believes this was attempt 1". On subsequent grades in this function, a regrade is called, which should not increment the attempt number, and hence the context should still think it's attempt 1.